apps/kg/eval.py

```diff
@@ -98,10 +106,14 @@ def main(args):
     # Here we want to use the regualr negative sampler because we need to ensure that
     # all positive edges are excluded.
     eval_dataset = EvalDataset(dataset, args)
+
     args.neg_sample_size_test = args.neg_sample_size
```

Comment:
This var is not used in the following code.
Should we use it in creating test_sampler_head and test_sampler_tail sampler?